Tools/autotest/arduplane.py

```diff
@@ -5533,6 +5531,7 @@ def RunMissionScript(self):
 
     def MAV_CMD_GUIDED_CHANGE_ALTITUDE(self):
         '''test handling of MAV_CMD_GUIDED_CHANGE_ALTITUDE'''
+        self.start_subtest("set home relative altitude")
         self.takeoff(30, relative=True)
         self.change_mode('GUIDED')
         for alt in 50, 70:
```

Comment:
Can we also test negative MSL altitudes in death valley? You mentioned it didn't work before, so now we can prove it does work.

Reply:
I haven't tested this, follow up?